test/libsolidity/smtCheckerTests/math/addmod_1.sol

```diff
@@ -12,10 +12,5 @@ contract C {
 	}
 }
 // ----
-// Warning 1218: (83-108): CHC: Error trying to invoke SMT solver.
-// Warning 1218: (141-166): CHC: Error trying to invoke SMT solver.
-// Warning 1218: (76-114): CHC: Error trying to invoke SMT solver.
-// Warning 1218: (170-184): CHC: Error trying to invoke SMT solver.
-// Warning 1218: (263-278): CHC: Error trying to invoke SMT solver.
 // Warning 3046: (141-166): BMC: Division by zero happens here.
 // Warning 3046: (263-278): BMC: Division by zero happens here.
```

Comment:
I wonder why CHC still doesn't detect the division by 0 in `g`.

Reply:
Not sure, could be a next thing to investigate?